src/lwc/bdiFieldMappingModalComboboxSearch/bdiFieldMappingModalComboboxSearch.html

```diff
@@ -0,0 +1,63 @@
+<template>
```

Comment:
Is this component tied specifically to bdi funcationality or the modal specifically? I'm wondering if this could be a "util" component for more generic reuse. Not if that means refactoring though.

Reply:
This could be a util component. It isn't necessarily specific to bdi or the modal.